.github/workflows/build.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
         java-version: "${{ matrix.java }}"
     - uses: "bufbuild/buf-setup-action@v1.7.0"
```

Comment:
Should we instead use our centralized buf action?

Reply:
The shared action also generates assuming a buf.gen.yaml which isn't used in the Java client.